src/client/datascience/notebook/contentProvider.ts

```diff
@@ -44,16 +44,15 @@ export class NotebookContentProvider implements VSCNotebookContentProvider {
         if (!this.compatibilitySupport.canOpenWithVSCodeNotebookEditor(uri)) {
             // If not supported, return a notebook with error displayed.
             // We cannot, not display a notebook.
+            const cellData = new NotebookCellData(
```

Comment:
It's funny I thought that typescript had some syntactic sugar for this, like a C# field initializer syntax, but didn't see it.